protocol/common/rpcconsumerlogs.go

```diff
@@ -110,9 +118,42 @@ func (pl *RPCConsumerLogs) LogStartTransaction(name string) {
 	}
 }
 
-func (pl *RPCConsumerLogs) AddMetric(data *metrics.RelayMetrics, isNotSuccessful bool) {
+func (pl *RPCConsumerLogs) AddMetric(data *metrics.RelayMetrics, isSuccessful bool) {
 	if pl.StoreMetricData {
-		data.Success = !isNotSuccessful
+		data.Success = isSuccessful
 		pl.MetricService.SendData(*data)
 	}
 }
+
+func (pl *RPCConsumerLogs) ShouldCountMetricForHttp(c *fiber.Ctx) bool {
+	refererHeaderValue := c.Get(refererHeaderKey, "")
```

Comment:
why do we need to test both lower case and upper case? I wouldn't do both of them every time. 

can we decide on one of them that is valid and the other isn't?